bin/remove-all-images.sh

```diff
@@ -16,7 +16,7 @@
 for i in \
 restore pgdump pgrestore postgres-gis prometheus grafana collect pgbadger pgpool \
 watch backup postgres pgbouncer pgadmin4 scheduler upgrade backrest-restore pgbench \
-pgbasebackup-restore
+pgbasebackup-restore postgres-ha
```

Comment:
@andrewlecuyer crunchy-postgres-gis-ha too?

Reply:
`crunchy-postgres-gis-ha` is now included